mozilla/firefox-ios
# Bug 1223403 move to using dynamic frameworks

This moves all Carthage dependencies out of ThirdParty and subprojects to be dynamic linked frameworks.

To test, 
ensure you are on the latest version of cathage
`brew upgrade carthage`

checkout branch, delete Carthae/Checkouts/google-breakpad-ios and then run

`carthage update --platform ios --no-use-binaries`

Then clean and build Client.xcodeproj

You have to delete breakpad as there is an incompatibility between the carthage compatible version and the existing version.

@sleroux I had to revert to using the original patch of breakpad that moved the client.xcodeproj into the root as the workspace version did not compile consistently. Am still working on a better solution for this. View that part as temporary
